core/server/api/index.js

```diff
@@ -109,46 +108,6 @@ cacheInvalidationHeader = function cacheInvalidationHeader(req, result) {
     }
 };
 
-/**
- * ### Location Header
- *
- * If the API request results in the creation of a new object, construct a Location: header which points to the new
- * resource.
- *
- * @private
- * @param {Express.request} req Original HTTP Request
- * @param {Object} result API method result
- * @return {String} Resolves to header string
- */
-locationHeader = function locationHeader(req, result) {
```

Comment:
See https://github.com/TryGhost/Ghost/issues/9326#issuecomment-368844504. I am currently not sure about removing the `Location` header.